src/Native/LibMFWrapper/copyright.txt

```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2014-2015 The LIBMF Project.
```

Comment:
>2014-2015 [](start = 14, length = 9)

legalese question. 
Do we need to do something about `2014-2015`?
Also, do we need to put something out to the overall 'external attributions' somewhere, in addition to this file?

Reply:
We shouldn't modify the copyright document from the external package. For your second question, I can add some comments on C# side.